File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int count = words.length;

        for (String word : words) {
            for (int j = 0; j < word.length(); j++) {
                if (allowed.indexOf(word.charAt(j)) == -1) { // check if char is not in allowed
                    count--;
                    break;
                }
            }
        }

        return count;
    }
}

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        count = len(words)
        for i in words:
            for j in range(len(i)):
                if i[j] not in allowed:
                    count -= 1
                    break
        return count
        
        